src/librbd/operation/RebuildObjectMapRequest.cc

```diff
@@ -24,6 +24,8 @@
 namespace librbd {
 namespace operation {
 
+using util::create_context_callback;
```

Comment:
Nit: why not use the helper wrapper ```create_context_callback```?

Reply:
I missed this comment when fixing -- do you mean reuse `create_callback_context` (from `AsyncRequest`)?

Reply:
Whoop -- this comment was a mistake. I just searched for "using util::" (looking for ```util::detail::C_RefRequestCallback```) and I accidentally added the comment to this line.